docs/Shadows.mdx

```diff
@@ -0,0 +1,79 @@
+import { Meta, Canvas, Story } from '@storybook/addon-docs';
+import LinkTo from '@storybook/addon-links/react';
+
+<Meta title="Design Tokens/Shadows" />
+
+# Shadows
+
+Shadows convey elevation of elements on a surface
+
+## Size
+
+There are 4 different sizes of shadow in MetaMask.
+
+<Canvas>
+  <Story id="shadows-shadows--size" />
+</Canvas>
+
+| Size   | JS                | CSS                     |
```

Comment:
Do these token sizes and colors align in figma?

Reply:
Great question these should align with the tokens in Figma yes. I think Figma library is currently out of date @Akatori-Design